README.md

```diff
@@ -49,7 +49,20 @@ export $(cat .env | xargs)  # If you want to set or update the current shell env
 java -jar app.jar --spring.main.web-application-type=none --import-locations-and-prices
 ```
 
-Importing supplier reports for given dates:
+
+Importing supplier reports with no supplied dates:
+
+*Note: using this defaults to one days worth of report data ending on the previous day to today's date.  If the current date is 30th November then from and to would be from 28th to 29th November.*
+
+```bash
+export $(cat .env | xargs)  # If you want to set or update the current shell environment
+
+java -jar app.jar --spring.main.web-application-type=none --import-supplier-reports
+```
+
+Importing supplier reports with supplied dates:
+
+*Note: care should be taken when supplying the date range.  This should not go beyond a month period due to the volume of reporting data*
```

Comment:
At the moment it unfortunately will need to be set for longer than a month period, because of the backfill issue. Data for a move can currently span months, which is a problem. That's one of the reasons I'm thinking we'll need to do incremental imports, persisting the reporting data for each move.